Themis/API/ThemisAPI.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ThemisAPI.swift
+//  Themis
+//
+//  Created by Andreas Cselovszky on 25.01.23.
+//
+
+import Foundation
+import CodeEditor
+
+enum ThemisAPI {
```

Comment:
Maybe call this `ThemisMLAPI` to emphasize that it's different from Artemis? Not sure though, those uppercase letters look a bit ugly as well

Reply:
ThemisMLAPI looked very weird to me. Lets leave it like that and change it later if needed.